allensdk/brain_observatory/vbn_2022/metadata_writer/dataframe_manipulations.py

```diff
@@ -334,6 +344,18 @@ def _patch_date_and_stage_from_pickle_file(
                 behavior_df.behavior_session_id == beh_id,
                 columns_to_patch] = new_row
 
+            if (beh_ct + 1) % n_to_log == 0 and logger is not None:
```

Comment:
Please use tqdm for a progress bar instead of rolling your own

Reply:
I actually don't like the output of `tqdm` for log files. As an example, I ran

```
from allensdk.brain_observatory.behavior.behavior_project_cache.\
    behavior_project_cache import VisualBehaviorOphysProjectCache

cache_dir = 'dummy_cache'
cache = VisualBehaviorOphysProjectCache.from_s3_cache(cache_dir=cache_dir)
```
(our cloud cache API uses tqdm to report the status of downloading the metadata files) and piped the output to a text file. Here is what I see when I examine that log file with `more`

<img width="570" alt="Screen Shot 2022-05-27 at 9 04 32 AM" src="https://user-images.githubusercontent.com/1682854/170736788-49f1e34d-51f5-4490-bf5c-43b616913eda.png">

I'd rather just leave this as is so that we have the option of storing the output to a log file.